packages/itwinui-css/backstop/tests/button.html

```diff
@@ -1248,14 +1577,22 @@ <h3>Borderless</h3>
 
     <h3>Split menu</h3>
     <section id="demo-split-menu">
```

Comment:
Nitpicking here, but should we change this id to `demo-split` if we're changing `iui-button-split-menu` to `iui-button-split`?

Reply:
For now I've only updated the test names & test ID.

Reply:
I would say either keep it at `Split menu` or just simply change it to `Split` to be consistent with other sections (Default, Borderless, etc.)

Reply:
If I make this change I will make it very last so as to not introduce test image changes.